src/main/java/com/example/gildedrose/service/ItemRepository.java

```diff
@@ -21,4 +22,10 @@ public Updateable save(Updateable item) {
     public ArrayList<Updateable> findAll() {
         return this.items;
     }
+
+    public void update() {
+        ArrayList<Updateable> updated = GildedRose.updateQuality(items);
```

Comment:
**question** Should a change in the `GildedRose` force a change in the `ItemRepository`?

Another way to ask this question is should the `ItemRepository` depend on the `GildedRose`? Or should the database depend on the business logic?

IMO, the database (or `ItemRepository`) is a detail of our application, what we really care about is the business logic (`GildedRose`). 

How could we refactor so that we maintain that the database is a detail?